src/configuration/ConfigurationManager.ts

```diff
@@ -15,7 +16,7 @@ export class ConfigurationManager {
             slotsInEpoch: Number(process.env.NUM_SLOTS_IN_EPOCH),
             commissionRatesByPublicKey: await getComissionRates(),
             stakingPoolPublicKey: process.env.POOL_PUBLIC_KEY || '',
-            payoutMemo: process.env.POOL_MEMO || 'mina-pool-payout',
+            payoutMemo: getMemoMd5Hash(process.env.POOL_PUBLIC_KEY || ''),
```

Comment:
I could see a common case where the pool wants its own memo e.g. Pool Payout Epoch 66, but for the burn addresses it uses this format.

Reply:
@garethtdavies payoutMemo will be used for both payouts and burn. May be I can make it more clear to avoid confusions.

Reply:
I think the point was to separate them - if MF requires the md5 hash memo for their program, it should be a specific memo, not take away the ability for people to use a different memo for the rest of the payouts. Many pools send a message like Gareth mentioned to make it clear for their stakers - let's separate the payoutMemo for MF (which could be used for the burn and for MF returns.)

Reply:
Change to this but I'm conscious this is not optimal:
payoutMemo: process.env.POOL_MEMO || getMemoMd5Hash(process.env.POOL_PUBLIC_KEY || ''),